pom.data.xml

```diff
@@ -19,7 +19,7 @@
   <parent>
     <groupId>com.azure</groupId>
     <artifactId>azure-sdk-parent</artifactId>
-    <version>1.2.0</version>
+    <version>1.1.0</version>
```

Comment:
This should still be 1.2.0 you need to update the version of this artifact above.

Reply:
Oh yeah, the parent section should move on top.
I just assumed the bottom one is azure-data-sdk-parent.
Updated the azure-data-sdk-parent verison.